nx-dev/feature-doc-viewer/src/lib/sidebar.tsx

```diff
@@ -63,7 +66,7 @@ export function Sidebar({
             onChange={(item) =>
               router.push(
                 createNextPath(item.value, flavor.alias, router.asPath)
-              )
+              ) && setStoredVersion(item.value)
```

Comment:
I'd write this as:

```
onChange={(item) => {
  router.push(
    ceateNextPath(item.value, flavor.alias, router.asPath)
  );
  setStoredVersion(item.value);
}
```

Or maybe better yet:

```
  router.push(
    ceateNextPath(item.value, flavor.alias, router.asPath)
  ).then(success => success && setStoredVersion(item.value))
```

Since the `.push` method returns a boolean indicating whether navigation succeeded or not.